src/DynamoUtilities/PathHelper.cs

```diff
@@ -110,7 +110,6 @@ public static bool isValidXML(string path, out XmlDocument xmlDoc, out Exception
             }
             catch(Exception e)
             {
-                Console.WriteLine(e.ToString());
```

Comment:
Why is this no longer required?

Reply:
@QilongTang are you okay with this being removed - was there a reason we wanted this?

Reply:
@mjkkirschner It was for diagnosing old corrupted XML, I can see it less useful now. Fine with this change

Reply:
👍 Thanks